apps/src/code-studio/pd/professional_learning_landing/SelfPacedProgressTable.jsx

```diff
@@ -41,6 +41,7 @@ const CourseRow = ({
             {percent_completed}% {i18n.selfPacedPlCompleted()}
           </BodyThreeText>
           {/* Progress bar */}
+          {/* eslint-disable-next-line react/forbid-dom-props */}
           <div className={styles.progressBar} data-testid="progress-bar">
```

Comment:
conceptually, I'm on board with the general idea that the test should be looking for elements that are accessible. Even after reading the linked [doc](https://testing-library.com/docs/queries/about/#priority') I'm a little fuzzy on what the fix would be. For example, in this case, it doesn't have a role or label text... is the better approach to give this element an aria-label and then `getByAltText`?

Reply:
yes, giving the element an `aria-label` and then calling `getByLabelText` would be perfect, and exactly the kind of fix that would make our site more accessible!